samples/snippets/csharp/api/system.reflection.metadata/metadatareader/MetadataReaderSnippets.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+
+namespace MetadataReaderSnippets
+{
+    class MetadataReaderSnippets
+    {
+        public static void Run()
+        {
+            //<SnippetMetadataReader>
+            FileStream fs = new FileStream("Example.dll", FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
```

Comment:
this could be directly in the using

Reply:
Good point. Moved into using in both cases.